youtube_dl/extractor/philharmonie.py

```diff
@@ -0,0 +1,45 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class PhilharmonieTvIE(InfoExtractor):
+    _VALID_URL = r'http://www\.philharmonie\.tv/veranstaltung/(?P<id>.*?)/'
+    _TEST = {
+        'url': 'http://www.philharmonie.tv/veranstaltung/26/',
+        'md5': '494b1e6605767f0d5bb141f879de1813',
+        'info_dict': {
+            'id': '26',
+            'ext': 'flv',
+            'title': u'03.05.2016 Tuesday 20:00 Uhr Konzert vom 03.05. ACHT BR\xdcCKEN | Musik f\xfcr K\xf6ln.  Hauschka, G. Schwellenbach, D. Brandt, P. Frick, E. Sarp, J. Farah: Reich',
+            'description': 'Trailer zum Livestram am 03.05.2016 Tuesday 20:00h, Konzert vom 03.05. ACHT BRÜCKEN | Musik für Köln.  Hauschka, G. Schwellenbach, D. Brandt, P. Frick, E. Sarp, J. Farah: Reich',
+        }
+    }
+
+    IE_NAME = 'philharmonie.tv'
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id)
+        source_url = self._search_regex(
+            r'<iframe id="vidframe".*?data-src="(https://playout.3qsdn.com/[\w-]+)"',
```

Comment:
It's a generic embed. It should be handled by generic extractor that should delegate extraction to separate 3qsdn extractor.